src/providers/saml-jackson.ts

```diff
@@ -0,0 +1,51 @@
+import type { OAuthConfig, OAuthUserConfig } from "."
+
+export interface SAMLJacksonProfile {
+  id: string
+  email: string
+  firstName: string
+  lastName: string
+  name: string
+  email_verified: boolean
+  image: null
+}
+
+export default function SAMLJackson<
+  P extends Record<string, any> = SAMLJacksonProfile
+>(options: OAuthUserConfig<P>): OAuthConfig<P> {
+  return {
+    id: "saml-jackson",
+    name: "BoxyHQ SAML Jackson",
+    type: "oauth",
+    version: "2.0",
+    checks: ["pkce", "state"],
+    authorization: {
+      url: `${options.issuer}/api/oauth/authorize`,
+      params: {
+        scope: "",
+        response_type: "code",
+        provider: "saml",
+      },
+    },
+    token: {
+      url: `${options.issuer}/api/oauth/token`,
+      params: { grant_type: "authorization_code" },
+    },
```

Comment:
```suggestion
    token: `${options.issuer}/api/oauth/token`,
```

I believe that parameter is added by default, no need for you to specify. But please check.